ludwig/api.py

```diff
@@ -1438,26 +1439,10 @@ def save_config(self, save_path: str) -> None:
         model_hyperparameters_path = os.path.join(save_path, MODEL_HYPERPARAMETERS_FILE_NAME)
         save_json(model_hyperparameters_path, self.config)
 
-    def save_savedmodel(self, save_path: str) -> None:
-        """This function allows to save models on disk.
-
-        # Inputs
-
-        :param  save_path: (str) path to the directory where the SavedModel
-                is going to be saved.
-
-        # Return
-
-        :return: `None`
-
-        # Example usage
-
-        ```python
-        ludwig_model.save_for_serving(save_path)
-        ```
-        """
+    def to_torchscript(self):
         self._check_initialization()
-        self.model.save_savedmodel(save_path)
+        inf_model = InferenceModule(self.model, self.config, self.training_set_metadata)
```

Comment:
nit: Could we use "inference_module" here? Minor preference for spelled out variable names.